### PR TITLE
Add turkish translation

### LINKED ISSUE #16755: Change request of bootstrap/docs/_data/translations.yml
We have translated the whole documentation into Turkish. We request you to add a link into "Translations" part of "Getting Started" page.
You can reach our web site: www.trbootstrap.com
Regards,
Meftun.

### PR DESCRIPTION
**Pull request by kkirsche**
Fix #16755

### CODE REVIEW

File: docs/_data/translations.yml [hunk 1]
```diff
@@ -47,3 +47,8 @@
   code: vi
   description: Bootstrap bằng tiếng Việt
   url: http://getbootstrap.com.vn
+
+  - name: Turkish
```
Comment (cvrebert) [inline docs/_data/translations.yml]: this line should not be indented

Comment (cvrebert) [inline docs/_data/translations.yml]: Also, this entry should come after "Spanish" based on how we're alphabetizing these

**Comment by kkirsche** [comment]
@cvrebert Fixed. Sorry about those errors

**Comment by cvrebert** [comment]
Thanks

**Comment by kkirsche** [comment]
Of course. Happy to help out even if just in a little way

**Pull request merged**